test/test_model.py

```diff
@@ -715,12 +715,10 @@ def test_storage(self):
         self.assertEqual(len(model.storages), 2)
         self.assertEqual(model.storages.keys(), meta.storages.keys())
         self.assertIn('disks', model.storages)
-        try:
+
+        with self.assertRaises(KeyError) as cm:
             model.storages['does-not-exist']
-        except KeyError as err:
-            assert 'Did you mean' in str(err), 'got wrong error message'
-        except Exception as err:
-            assert False, 'got wrong exception type: ' + str(err)
+        assert 'Did you mean' in str(cm.exception)
```

Comment:
as per https://docs.pytest.org/en/6.2.x/assert.html:

You can pass a match keyword parameter to the context-manager to test that a regular expression matches on the string representation of an exception.

```
with self.assertRaises(KeyError, match='Did you mean') as cm:
    model.storages['does-not-exist']
```

Reply:
done